packages/react/src/components/Authenticator/SignUp/FormFields.tsx

```diff
@@ -70,15 +76,29 @@ export function FormFields() {
             );
           case 'password':
             return (
-              <PasswordField
-                key={name}
-                autoComplete="new-password"
-                data-amplify-password
-                {...propsCreator('password', 'Password', formOverrides, true)}
-                hasError={!!validationError['confirm_password']}
-                name="password"
-                onBlur={handleBlur}
-              />
+              <React.Fragment key={name}>
+                <PasswordField
+                  autoComplete="new-password"
+                  data-amplify-password
+                  {...propsCreator('password', 'Password', formOverrides, true)}
+                  hasError={!!validationError['confirm_password']}
+                  name="password"
+                  onBlur={handleBlur}
+                />
+                {validationError['password'] && (
```

Comment:
nit: Do we have a preference in this codebase for bracket notations vs dot notations?

Reply:
I'm ok with either. We don't have a style guide on this.